src/main/resources/i18n/messages.properties

```diff
@@ -481,21 +481,22 @@ user.create.email.taken=This email is already taken. Please supply a different e
 user.create.phone.no-numbers=Phone number must contain numbers
 user.setpassword=Require User Activation
 user.edit.details=Edit User Details For {0}
-user.edit.password.match=Password and Confirm Password do not match.
+user.edit.password.old.incorrect=The Old Password is incorrect
+user.edit.password.new.match=New Password and Confirm New Password do not match.
```

Comment:
Change this message to just `Passwords do not match`

Reply:
Please see 453b0ef8bc524172456e276fddb553346c3c3cd8.